src/sentry/static/sentry/app/views/projectReleases/index.jsx

```diff
@@ -18,7 +18,7 @@ import {Panel, PanelBody, PanelHeader} from 'app/components/panels';
 import EmptyStateWarning from 'app/components/emptyStateWarning';
 
 import ReleaseList from 'app/views/projectReleases/releaseList';
-
+import ReleaseLanding from 'app/views/projectReleases/releaseLanding.jsx';
```

Comment:
nit: remove file extension in imports

Reply:
Can we automate this?

Reply:
Yes it's on the list: https://github.com/getsentry/eslint-config-sentry/issues/6